mlos_bench/mlos_bench/config/environments/os/linux/runtime/linux-runtime-tunables.jsonc

```diff
@@ -1,14 +1,66 @@
 {
-    "linux-kernel-runtime": {
+    "linux-scheduler": {
         "cost": 1,
-        // TODO: FIXME: These should include actual filesystem paths to the parameters.
         "params": {
-            "kernel_sched_migration_cost_ns": {
+            "sched_migration_cost_ns": {
                 "description": "Cost of migrating the thread to another core",
                 "type": "int",
-                "default": -1,
-                "range": [-1, 500000],
+                "meta": {"name_prefix": "/proc/sys/kernel/"},
```

Comment:
Future work we already discussed elsewhere, but I want to revist how we handle these prefixes.

Is there a reason we don't put it directly in the name atm?

Reply:
yes, since we can have the tunables in the shell env variables, the names must be valid identifiers - or else we'll apply some normalization rules to them. so I thought it would be easier to move that info to the metadata. OTOH, maybe we should have something like `"meta": {"name": "/proc/sys/kernel/sched_migration_cost_ns"}` instead? We can add that in the next PR

Reply:
Yeah, I guess we can play with it.  As mentioned elsewhere, for some paths it ends up creating conflicts if the filename is common, but the path is unique.
We also had some logic for doing param name mapping for environment variables and made sure to only explicitly expose requested params as environment variables, so I don't think that's too much of a concern here.

Reply:
I guess my main question was if you ran into something that treated `/` characters specially (I have in the past, but that was a different optimizer and configspace we aren't supporting anymore).